notebooks/Model_Training_with_BERT.ipynb

```diff
@@ -0,0 +1,5371 @@
+{
```

Comment:
<p>Can you move the text of this comment to the markdown cell above and correct the spelling of "convenience"?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CODAIT/text-extensions-for-pandas/pull/116/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>